react/features/base/i18n/BuiltinLanguages.native.js

```diff
@@ -0,0 +1,200 @@
+import i18next from 'i18next';
+
+import BG_MAIN from '../../../../lang/main-bg';
+import BG_LANG from '../../../../lang/languages-bg';
+
+import DE_MAIN from '../../../../lang/main-de';
+import DE_LANG from '../../../../lang/languages-de';
+
+import EO_MAIN from '../../../../lang/main-eo';
+import EO_LANG from '../../../../lang/languages-eo';
+
+import ES_MAIN from '../../../../lang/main-es';
+import ES_LANG from '../../../../lang/languages-es';
+
+import FR_MAIN from '../../../../lang/main-fr';
+import FR_LANG from '../../../../lang/languages-fr';
+
+import HY_MAIN from '../../../../lang/main-hy';
+import HY_LANG from '../../../../lang/languages-hy';
+
+import IT_MAIN from '../../../../lang/main-it';
+import IT_LANG from '../../../../lang/languages-it';
+
+import NB_MAIN from '../../../../lang/main-nb';
+import NB_LANG from '../../../../lang/languages-nb';
+
+import OC_MAIN from '../../../../lang/main-oc';
+import OC_LANG from '../../../../lang/languages-oc';
+
+import PL_MAIN from '../../../../lang/main-pl';
+import PL_LANG from '../../../../lang/languages-pl';
+
+import PTBR_MAIN from '../../../../lang/main-ptBR';
+import PTBR_LANG from '../../../../lang/languages-ptBR';
+
+import RU_MAIN from '../../../../lang/main-ru';
+import RU_LANG from '../../../../lang/languages-ru';
+
+import SK_MAIN from '../../../../lang/main-sk';
+import SK_LANG from '../../../../lang/languages-sk';
+
+import SL_MAIN from '../../../../lang/main-sl';
+import SL_LANG from '../../../../lang/languages-sl';
+
+import SV_MAIN from '../../../../lang/main-sv';
+import SV_LANG from '../../../../lang/languages-sv';
+
+import TR_MAIN from '../../../../lang/main-tr';
+import TR_LANG from '../../../../lang/languages-tr';
+
+import ZHCN_MAIN from '../../../../lang/main-zhCN';
+import ZHCN_LANG from '../../../../lang/languages-zhCN';
+
+
+/**
+ * Collection of builtin languages.
+ */
+const languages = [
+
+    // Bulgarian
+    {
+        name: 'bg',
+        mainResource: BG_MAIN,
+        langResource: BG_LANG
```

Comment:
@saghul, I'm afraid I don't understand the purpose of `langResource`! I don't see it used anywhere (in the file given that `languages` is not exported out of the file).

Reply:
If `langResource` is not necessary, could we please maybe simplify the source code? Like maybe converting the `languages` array to a map, the `name` to a key in the map, and `mainResource` to the value associated with the key that's `require`?

````javascript
const languages = {
    bg: require('../../../../lang/main-bg')
}
````

Reply:
It is necessary, I just made a mistake :-( it should go here: https://github.com/jitsi/jitsi-meet/pull/2318/files#diff-c2a31f269ba5f30dfabe9049972beafaR196

Reply:
I'll fix that, and see if we can make this any simpler. 👍